CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Lotus changelog
 
+# miner/v1.28.2 / 2024-08-14
+
+This is a Lotus-Miner patch release that fixes the pledge issues users have been encountering. This update addresses existing issues, including the too-small pledge in snap and the lack of DDO-awareness in PoRep Commit.
```

Comment:
Is there an issue to link for this?

Reply:
Not that I am aware of. @filecoin-project/curio might have insights if there is any open issues for the pledge issues?